docs/iot-gateway/config/opc-ua.md

```diff
@@ -10,410 +8,242 @@ description: OPC-UA protocol support for ThingsBoard IoT Gateway
+Unified Architecture) is a machine-to-machine communication protocol for industrial automation, and this connector 
+allows seamless integration with the ThingsBoard platform.
+Use [general configuration](/docs/iot-gateway/configuration/) to enable this extension.
+
+We will describe the connector configuration below.
+
 {% capture difference %}
-**Starting from Gateway version 3.1, we added a new OPC-UA connector based on the AsyncIO library. 
-Please note that the connector is in the early beta stage and may contain bugs. 
-Also, it is not recommended to use it in production mode for now.
-To enable it, use the type of connector "opcua_asyncio".**
+**Please note:**
+If you are new to IoT Gateway, use the "Basic" configuration mode. If you are familiar with configuring IoT Gateway, 
+you can use the "Advanced" configuration mode.
 {% endcapture %}
 {% include templates/info-banner.md content=difference %}
 
-This guide will help you to get familiar with OPC-UA connector configuration for ThingsBoard IoT Gateway.
-Use [general configuration](/docs/iot-gateway/configuration/) to enable this extension.
-The connector configuration file will be described below.
-
-<b>Example of OPC-UA Connector config file.</b>
-
-{% capture opcuaConf %}
-
-{
-  "server": {
-    "name": "OPC-UA Default Server",
-    "url": "localhost:4840/freeopcua/server/",
-    "timeoutInMillis": 5000,
-    "scanPeriodInMillis": 5000,
-    "disableSubscriptions":false,
-    "subCheckPeriodInMillis": 100,
-    "showMap": false,
-    "security": "Basic128Rsa15",
-    "identity": {
-      "type": "anonymous"
-    },
-    "mapping": [
-      {
-        "deviceNodePattern": "Root\\.Objects\\.Device1",
-        "deviceNamePattern": "Device ${Root\\.Objects\\.Device1\\.serialNumber}",
-        "attributes": [
-          {
-            "key": "CertificateNumber",
-            "path": "${ns=2;i=5}"
-          }
-        ],
-        "timeseries": [
-          {
-            "key": "temperature °C",
-            "path": "${Root\\.Objects\\.Device1\\.TemperatureAndHumiditySensor\\.Temperature}"
-          },
-          {
-            "key": "batteryLevel",
-            "path": "${Battery\\.batteryLevel}"
-          }
-        ],
-        "rpc_methods": [
-          {
-            "method": "multiply",
-            "arguments": [2, 4]
-          }
-        ],
-        "attributes_updates": [
-          {
-            "attributeOnThingsBoard": "deviceName",
-            "attributeOnDevice": "Root\\.Objects\\.Device1\\.serialNumber"
-          }
-        ]
-      }
-    ]
-  }
-}
+## Connector configuration
+
+The connector configuration is a user interface form that helps you set up a connection to OPC-UA server. 
+Let's look at all the available settings and explain each one clearly. This will help you understand how 
+everything works.
 
+{% capture difference %}
+**Please note:**
+Connect the gateway before creating the connector. Otherwise, the UI will display the old configuration fields 
```

Comment:
To get the actual UI for the gateway - you need to have connected gateway, before adding connector. Otherwise, you will see old UI.